web/src/components/ChatConversation.tsx

```diff
@@ -24,27 +24,26 @@ const ErrorSendingStatus = styled.div`
 `
 
 type ChatConversationProps = {
-  hasConversationStarted: boolean
   messages: ChatMessageModel[]
   hasError: boolean
   className?: string
 }
 
-const ChatConversation = ({
-  hasConversationStarted,
-  messages,
-  hasError,
-  className,
-}: ChatConversationProps): ReactElement => {
+const ChatConversation = ({ messages, hasError, className }: ChatConversationProps): ReactElement => {
   const { t } = useTranslation('chat')
+  const [messagesCount, setMessagesCount] = useState(0)
   const messagesEndRef = useRef<HTMLDivElement>(null)
 
-  const scrollToBottom = () => messagesEndRef.current?.scrollIntoView({ behavior: 'smooth' })
-  useEffect(scrollToBottom, [messages])
+  useEffect(() => {
+    if (messagesCount < messages.length) {
+      messagesEndRef.current?.scrollIntoView({ behavior: 'smooth' })
+      setMessagesCount(messages.length)
+    }
+  }, [messages, messagesCount])
 
   return (
     <Container className={className}>
-      {hasConversationStarted ? (
+      {messages.length > 0 ? (
```

Comment:
(minor) I liked this named constant

Reply:
The prop was redundant and a relict of old code. I therefore chose to remove it.